lib/tasks/convicts.rake

```diff
@@ -1,28 +1,18 @@
 namespace :convicts do
-  desc 'Link all convicts to their organizations'
```

Comment:
this rake was used to migrate from "under_hand_of" architecture to the organization one
it is irrelevant now that the migration is done